### PR TITLE
chore: update the output when listing files

### PR DESCRIPTION
**Pull request by uriii3**
From ticket [CMT-108](https://cms-change.atlassian.net/browse/CMT-108), were a get was being done and the toolbox was taking so long, we thought that it might be interesting to add, as an output, some sort of information about the process that is being done underneath.

This might help users understand that, although it will take a while, it will eventually reach and download the files.

### CODE REVIEW

File: copernicusmarine/download_functions/download_original_files.py [hunk 1]
```diff
@@ -468,11 +467,10 @@ def _list_files_on_marine_data_lake_s3(
         Prefix=prefix,
         Delimiter="/" if not recursive else "",
     )
-
+    logger.info("Listing files on the server...")
```
Comment (renaudjester) [inline copernicusmarine/download_functions/download_original_files.py:470]:
```suggestion
    logger.info("Listing files on remote server...")
```

**Review by renaudjester — APPROVED** [review]

File: copernicusmarine/download_functions/download_original_files.py [hunk 2]
```diff
@@ -468,11 +467,10 @@ def _list_files_on_marine_data_lake_s3(
         Prefix=prefix,
         Delimiter="/" if not recursive else "",
     )
-
+    logger.info("Listing files on remote server...")
     s3_objects = chain(
-        *map(lambda page: page.get("Contents", []), page_iterator)
+        *map(lambda page: page.get("Contents", []), tqdm(page_iterator))
```
Comment (renaudjester) [inline copernicusmarine/download_functions/download_original_files.py:472]: need to be disabled with the disable progress bar

File: copernicusmarine/download_functions/download_original_files.py [hunk 3]
```diff
@@ -468,11 +475,13 @@ def _list_files_on_marine_data_lake_s3(
         Prefix=prefix,
         Delimiter="/" if not recursive else "",
     )
-
+    logger.info("Listing files on remote server...")
```
Comment (uriii3) [inline copernicusmarine/download_functions/download_original_files.py:478]: what do you think, should I also delete this print if the flag `disable-progress-bar` is activated? for me it makes more sense maybe, no? to not have anything if they disable the progress bar

Comment (uriii3) [reply copernicusmarine/download_functions/download_original_files.py:478]: Mmh okay, alright! I will merge when it passes the tests then!

**Pull request merged**